Mathlib/Analysis/SpecialFunctions/Log/Basic.lean

```diff
@@ -324,10 +324,12 @@ theorem continuousOn_log : ContinuousOn log {0}ᶜ := by
   conv in log _ => rw [log_of_ne_zero (show (x : ℝ) ≠ 0 from x.2)]
   exact expOrderIso.symm.continuous.comp (continuous_subtype_val.norm.subtype_mk _)
 
+/-- The real logarithm is continuous as a function from nonzero reals. -/
 @[continuity]
```

Comment:
As I argue above, I suspect this is not useful (but I'm happy to be proven wrong).
```suggestion
```